packages/loader/container-loader/src/container.ts

```diff
@@ -978,6 +980,18 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this.emit("warning", warning);
     }
 
+    public getPackageName(codeDetails: IFluidCodeDetails | undefined): IContainerPackageInfo {
```

Comment:
This should be private unless we have an explicit need to expose it.

Reply:
The only thought behind keeping it public was if other partner repos want to use it instead of having the same code all over again.

Reply:
Let's wait to hear the request before exposing publicly -- small duplication is maybe annoying but much cheaper than long-term support of a public API.  Also, if they want to access it on the container then we'll need to add it to `IContainer` as well, so there is more work to do.

Reply:
Actually you can also pull this out to be a standalone helper function (rather than a member of `Container` since it doesn't require access to any members of the `Container`.  This will help keep `Container`'s scope smaller, plus if we want to expose the helper publicly in the future it would probably be better to do it as a standalone helper.

Reply:
I was initially thinking of having this in `fluidPackage#container-definitions` itself but wasn't sure how to release a new version of container-definitions.

Reply:
Yes - this should be a helper function outside of container. It's not functionality of the container, rather it relates to code details.

Reply:
It should stay in container-loader (the *-packages files should have no runtime code, just TS types) but within the container-loader package it can move wherever would make the most sense.

Reply:
Moved the helper function to contracts.ts#container-loader